client/src/components/UserCursors.tsx

```diff
@@ -181,6 +186,18 @@ export const UserCursor = React.forwardRef(
 							display: 'flex'
 						}}
 					>
+						{/* {add weather state here} */}
+
+						{weather!.temp.length > 0 ? (
```

Comment:
confusing syntax, instead of writing an empty div like ` : ( <div></div>)` you can conditionally render a component like this:
```{weather && weather.temp.length > 0 && <div> <div>{weather.temp} etc... </div>}```